From b949400a88d13c98e6cd30ffa47b30324505ab23 Mon Sep 17 00:00:00 2001
From: Benjamin Raison <benjamin.raison@baumink.net>
Date: Sat, 10 Jun 2023 00:34:28 +0200
Subject: [PATCH] hwdb: fix arrow keys on HP Elite Dragonfly G3

(cherry picked from commit 4fd7c712dcba3c4ed7183ba327d0b88d9b0be9bb)

Related: RHEL-5950
---
 hwdb.d/60-keyboard.hwdb | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index 27113276c4..492092e685 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -827,6 +827,13 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHP*:pnHPEliteDragonflyG2*:pvr*
 evdev:name:Intel HID events:dmi:bvn*:bvr*:bd*:svnHP*:pnHPEliteDragonflyG2*:pvr*
  KEYBOARD_KEY_08=unknown                                # rfkill is also reported by HP Wireless hotkeys
 
+# HP Elite Dragonfly G3
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHP:pnHPEliteDragonfly13.5inchG3NotebookPC:pvr*
+ KEYBOARD_KEY_c9=up
+ KEYBOARD_KEY_d1=down
+ KEYBOARD_KEY_c8=pageup
+ KEYBOARD_KEY_d0=pagedown
+
 ##########################################################
 # Huawei
 ##########################################################
